docs/source/guides/configuration.rst

```diff
@@ -38,6 +40,11 @@ Deployment-specific parameters
   username, password and database name.
   All data except the database name are mandatory.
   If database name is not provided, a default one gets used.
+- ``working database`` is the name of working database - the one that will be used to store the world state view and optionally blocks.
+    .. warning:: Deprecated behaviour: is overrided by ``dbname`` parameter in ``pg_opt`` value.
+    .. warning:: Deprecated: optional.
+- ``maintenance database`` is the name of databse that will not be used to store any data, but only to maintain the working database.
```

Comment:
```suggestion
- ``maintenance database`` is the name of a database that will not be used to store any data, but only to maintain the working database.
```

Reply:
The description does not provide any explanations for a user(((

What are those "maintenance needs"? Why I, as a user of Iroha, should be concerned with multiple databases creation while all the other apps use a single database.

Please accomplish the documentation with more explanation.


When the meaning is unclear, all of these look like unjustified complication. Very unfriendly.